### PR TITLE
Update docker instructions and docker-compose

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Fixes issue with database creation and starting containers

## Was the documentation (README, API, wiki, ...) updated?

README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ code will be shared into the container so that the webapp will be dynamically re
 [FIRST RUN]
 $ docker-compose up -d postgres
 $ docker-compose build app
+$ docker-compose build sidekiq
 $ docker-compose run app rake db:setup
```
Comment (jacobthill) [inline README.md]: do we need to add `bundle exec` to this line and line 84? I can't get it to work without that.

Comment (aaron-collier) [reply README.md]: oh good call. will update

Comment (aaron-collier) [reply README.md]: Added.

**Comment by jacobthill** [comment]
I'm getting the following error, even after pruning the docker container as recommended in the Readme: 
```Caused by:
PG::ConnectionBad: FATAL:  database "dlme" does not exist
/usr/local/bundle/ruby/2.6.0/gems/pg-1.2.2/lib/pg.rb:58:in `initialize'
/usr/local/bundle/ruby/2.6.0/gems/pg-1.2.2/lib/pg.rb:58:in `new'
/usr/local/bundle/ruby/2.6.0/gems/pg-1.2.2/lib/pg.rb:58:in `connect'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/postgresql_adapter.rb:692:in `connect'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/postgresql_adapter.rb:223:in `initialize'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/postgresql_adapter.rb:48:in `new'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/postgresql_adapter.rb:48:in `postgresql_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:830:in `new_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:874:in `checkout_new_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `try_to_checkout_new_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:814:in `acquire_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:538:in `checkout'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:382:in `connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_adapters/abstract/connection_pool.rb:1033:in `retrieve_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_handling.rb:118:in `retrieve_connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/connection_handling.rb:90:in `connection'
/usr/local/bundle/ruby/2.6.0/gems/activerecord-5.2.4.1/lib/active_record/model_schema.rb:324:in `table_exists?'
/opt/config/initializers/translations.rb:5:in `<main>'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:285:in `block in load'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:257:in `load_dependency'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:285:in `load'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/engine.rb:663:in `block in load_config_initializer'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/notifications.rb:170:in `instrument'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/engine.rb:662:in `load_config_initializer'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/engine.rb:620:in `block (2 levels) in <class:Engine>'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/engine.rb:619:in `each'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/engine.rb:619:in `block in <class:Engine>'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:50:in `each'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:50:in `tsort_each_child'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/application.rb:361:in `initialize!'
/opt/config/environment.rb:5:in `<main>'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/usr/local/bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:291:in `block in require'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:257:in `load_dependency'
/usr/local/bundle/ruby/2.6.0/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:291:in `require'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/application.rb:337:in `require_environment!'
/usr/local/bundle/ruby/2.6.0/gems/railties-5.2.4.1/lib/rails/application.rb:520:in `block in run_tasks_blocks'
/usr/local/bundle/ruby/2.6.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `load'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
/usr/local/bundle/gems/bundler-2.1.4/exe/bundle:46:in `block in <top (required)>'
/usr/local/bundle/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/usr/local/bundle/gems/bundler-2.1.4/exe/bundle:34:in `<top (required)>'
/usr/local/bundle/bin/bundle:23:in `load'
/usr/local/bundle/bin/bundle:23:in `<main>'
Tasks: TOP => db:setup => db:schema:load_if_ruby => db:create => db:load_config => environment
(See full trace by running task with --trace)```

**Comment by aaron-collier** [comment]
@jacobthill did you also prune the volume(s)? That's a required step as the existing database setup gets stored in a semi-persistent volume.

**Pull request merged**